algorithms/ar-soer/java/SieveOfEratosthenes.java

```diff
@@ -0,0 +1,43 @@
+//Sieve of Eratosthenes
+
+
+class SieveOfEratosthenes 
+{ 
+	void sieveOfEratosthenes(int number) 
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6371972)